src/basic/missing_ioprio.h

```diff
@@ -3,47 +3,77 @@
 
 #include <sched.h>
 
+#include "macro.h"
+
 /* Match values uses by the kernel internally, as no public header seems to exist. */
 
 #ifndef IOPRIO_N_CLASSES
 #  define IOPRIO_N_CLASSES 8
+#else
+assert_cc(IOPRIO_N_CLASSES == CLASSES 8);
 #endif
 
 #ifndef IOPRIO_BE_NR
 #  define IOPRIO_BE_NR 8
+#else
+assert_cc(IOPRIO_BE_NR == NR 8);
 #endif
 
 #ifndef IOPRIO_CLASS_NONE
 #  define IOPRIO_CLASS_NONE 0
+#else
+assert_cc(IOPRIO_CLASS_NONE == NONE 0);
```

Comment:
Hm??

Reply:
My patch was borked because I messed something up with the macro that was generating this. And then were were including an unrelated header, so we never triggered the paths with `assert_cc`. I added an extra commit to change the include, which causes a build failure with the old version of the patch, and then fixed the patch.